assets/js/coder.js

```diff
@@ -75,6 +75,17 @@ function setTheme(theme) {
         })
         
     }
+
+    function sendMessage(message) {
```

Comment:
The theme switch code is getting messier and messier. But, don't worry, we can refactor it later. :)

Reply:
Agreed, I noticed there are some repeated functions that can probably be refactored and condensed in the future to be cleaner. It's something that I would have definitely taken on if my JS knowledge was better.